klippy/stepper.py

```diff
@@ -94,6 +94,10 @@ def get_oid(self):
         return self._oid
     def get_step_dist(self):
         return self._step_dist
+    def set_step_dist(self, dist):
+        self._step_dist = dist
+        logging.info("%s manually set to =%.6f", (self._name, dist))
+        return self._step_dist
```

Comment:
This `return` statement looks redundant.

Reply:
Yes, should have trusted my gut on that one, added it thinking "this isn't useful at all"